recipes/sundials/bld.bat

```diff
@@ -0,0 +1,26 @@
+mkdir build
+cd build
+
+cmake ^
+    -G "%CMAKE_GENERATOR%" ^
+    -DCMAKE_PREFIX_PATH=%LIBRARY_PREFIX% ^
+    -DCMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
+
+    -DBUILD_BENCHMARKS=no ^
+    -DINTEGER_CLASS=gmp ^
+    -DWITH_SYMENGINE_THREAD_SAFE=yes ^
+    -DCMAKE_BUILD_TYPE=Release ^
+    -DBUILD_FOR_DISTRIBUTION=yes ^
+    -DBUILD_SHARED_LIBS=yes ^
+
+    -DCMAKE_BUILD_TYPE=Release ^
+    -DBUILD_SHARED_LIBS=1 ^
+    -DBUILD_STATIC_LIBS=0 ^
+    -DEXAMPLES_ENABLE=1 ^
+    -DEXAMPLES_INSTALL=0 ^
+    -DOPENMP_ENABLE=1 ^
+    -DLAPACK_ENABLE=0 ^
+    ..
+
+cmake --build . --config Release
+cmake --build . --config Release --target install
```

Comment:
Are there any tests that we can run on Windows? `ctest`? I am a zero on Windows so I cannot help you with that failure. Pinging @patricksnape, @msarahan, and @gillins who are our Windows heroes for a better opinion.

Note that we can merge it with that failure and fix in the feedstock.